### PR TITLE
Add pyscreeze to PyAutoGUI stub

### PR DESCRIPTION
**Pull request by Avasam**
Successor to #8823

### CODE REVIEW

**Comment by JelleZijlstra** [comment]
CI is failing because the pyscreeze package hasn't been uploaded yet, let's retry in a few hours.

The code looks good to me. Thanks for all your work contributing these stubs!

**Comment by AlexWaygood** [comment]
Need to wait a little bit until the typeshed bots have uploaded the first release of `types-pyscreeze` to PyPI :)

File: stubs/PyAutoGUI/pyautogui/__init__.pyi [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
-from collections.abc import Callable, Generator, Iterable, Sequence
+from collections.abc import Callable, Iterable, Sequence
 from datetime import datetime
-from typing import NamedTuple, SupportsFloat, SupportsInt, TypeVar, overload
+from typing import NamedTuple, SupportsInt, TypeVar
 from typing_extensions import ParamSpec, SupportsIndex, TypeAlias
 
-from PIL import Image
+from pyscreeze import (
+    Box as Box,
```
Comment (AlexWaygood) [inline stubs/PyAutoGUI/pyautogui/__init__.pyi:8]:
```suggestion
    Box,
```

File: stubs/PyAutoGUI/pyautogui/__init__.pyi [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
-from collections.abc import Callable, Generator, Iterable, Sequence
+from collections.abc import Callable, Iterable, Sequence
 from datetime import datetime
-from typing import NamedTuple, SupportsFloat, SupportsInt, TypeVar, overload
+from typing import NamedTuple, SupportsInt, TypeVar
 from typing_extensions import ParamSpec, SupportsIndex, TypeAlias
 
-from PIL import Image
+from pyscreeze import (
+    Box,
```
Comment (AlexWaygood) [inline stubs/PyAutoGUI/pyautogui/__init__.pyi:8]:
```suggestion
```

**Review by AlexWaygood — APPROVED** [review]
Thanks!

**Pull request merged**